src/app/modules/group/components/group-manager-list/group-manager-list.component.html

```diff
@@ -11,37 +11,78 @@
     [scrollable]="true"
     class="slanted-grid"
     >
-
-    <ng-template #headerTemplate let-columns>
-      <tr style="border: none;">
-        <th style="min-width: 110px;">
-          <div class="slanted-header">
-            <div class="slanted-header-content">
-              Name
+      <ng-template #headerTemplate let-columns>
+        <tr style="border: none;">
+          <th style="width: 160px;">
+            <div class="slanted-header">
+              <div class="slanted-header-content">
+                Name
+              </div>
             </div>
-          </div>
-        </th>
-        <th style="width: 160px; transform: none;">
-      </tr>
-    </ng-template>
-
-    <ng-template
-      #bodyTemplate
-      let-rowData
-      let-columns="columns"
-      let-rowIndex="rowIndex"
-    >
-      <tr [pSelectableRow]="rowData" [pSelectableRowIndex]="rowIndex">
-        <td
-          style="text-align: left;width: 160px;padding-left: 0.5rem;"
-          tooltipPosition="top"
-          tooltipStyleClass="tooltip-custom"
-        >
-          {{ rowData.name }}
-        </td>
+          </th>
+          <th style="min-width: 110px;">
+            <div class="slanted-header">
+              <div class="slanted-header-content">
+                Can Manage
+              </div>
+            </div>
+          </th>
+          <th style="width: 5rem;">
+            <div class="slanted-header">
+              <div class="slanted-header-content">
+                Can grant group access
+              </div>
+            </div>
+          </th>
+          <th style="width: 5rem;">
+            <div class="slanted-header">
+              <div class="slanted-header-content">
+                Can watch members
+              </div>
```

Comment:
The problem is that these labels go out of the screen and so are not visible. Maybe we could do it in the other direction? Could you try or suggest an idea?

Reply:
In the design, this is made possible by the last, column which is the 'Edit' column, that adds enough space for all the headers to be visible. For now I just added an empty column at the end (that will later be replaced by the 'edit' column)

Reply:
Ok thx. We will still have to fix the column repartition (80% is taken by the second column), but that's something else.